deployer-first-blocks.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:14.17
```

Comment:
To be consistent with the rest of the code, could we name this file `genesis-blocks-deployer` please?

Reply:
Done